dojo/api_v2/serializers.py

```diff
@@ -1327,16 +1327,16 @@ def validate(self, data):
 
     def validate_scan_data(self, value):
         # scan_date is no longer deafulted to "today" at import time, so set it here if necessary
-        if not value.date:
-            return None
+        # if not value.date:
+        #     return None
```

Comment:
I'm not a big fan of commented code

Reply:
Agreed, forgot to remove it

Reply:
Turns out this code is never executed: #5584